src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -91,11 +91,11 @@ export class ArticlesTable extends Component {
                     >
                       <MenuItem
                         disabled
-                        primaryText={`Saved: ${moment(item.get('timestamp')).format('lll')}`}
+                        primaryText={`Saved: ${moment.utc(item.get('timestamp')).local().format('lll')}`}
```

Comment:
You can send me a billion of links and articles about KISS and all these design patterns but I still don't see the reason to move it to separate module.

These 2 lines have the difference in position equals 3 lines. So, it's super simple to change them both, `moment.utc(x).local().format('lll')` is more readable and descriptive for me than `format_time_to_user`. Better to write unit test to not let developer change it accidentally to some other format, but what you are talking − it's to minor for me. I'd think about it later, when I see that this format is used in 2 different modules.